src/app/events/[eventId]/digitalWaiver/1/page.tsx

```diff
@@ -11,6 +11,8 @@ import styles from "./page.module.css";
 import beaverLogo from "/docs/images/beaver-logo.svg";
 import Image from "next/image";
 import NextLink from "next/link";
+import "../../../../fonts/fonts.css";
```

Comment:
oh no not the ../../../ lmfao, I'm assuming this import is to fix the current issues with other .css modifying fonts..